### DISCUSSION TITLE
Is it possible to make supabase realtime auth only?

### DISCUSSION THREAD

**Discussion by Aadim1** [supabase/supabase#19822] [Questions]
Is it possible to only allow connection to user who present a valid JWT secret i.e. authenticated user, and reject other connection.

**Comment by GaryAustin1** [comment]
Realtime has several modes.   Certainly with postgres changes if you have RLS select policy on the table and that limits to authenticated user then you get what you want.
I don't think the other modes have that feature but don't use them.
